odb_extractor_atoms.py

```diff
@@ -348,7 +781,7 @@ def seqxlsx(sequence, comment='', path=''):
         if 'GMOS' in inst:
             coadds = '1'
             # convert wavelength to microns
-    #             wavelength = '{:5.3f}'.format(float(wavelength) / 1000.)
+#             wavelength = '{:5.3f}'.format(float(wavelength) / 1000.)
         else:
             coadds = step['observe:coadds']
         data.append(int(coadds))
```

Comment:
Never mind: I got my answer here. 👍 
I think it makes more sense to just work with `int` here:
```
coadds = 1 if 'GMOS' in inst else int(step['observe:coadds'])
```
(or keep it as the structured `if` `else` if we are going to need the wavelength conversion).